.github/workflows/gradle-compatibility.yml

```diff
@@ -0,0 +1,48 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+
+name: Gradle Compatibility Tests
+on: [push, pull_request]
+jobs:
+  test:
+    name: "Compatibility: gradle ${{ matrix.gradle }}/java ${{ matrix.java }}"
+    runs-on: "ubuntu-latest"
+    strategy:
+      matrix:
+        avro: ["1.11.0"]
+        gradle: [
+          "5.1", "5.1.1", "5.2", "5.2.1", "5.3", "5.3.1", "5.4", "5.4.1", "5.5", "5.5.1", "5.6", "5.6.1", "5.6.2", "5.6.3", "5.6.4",
```

Comment:
Why v5.1 ? Isn't it too old ?
I am not Gradle user but AFAIK 8.x is the current version.
I ask because this PR increases the CI jobs for Java from less than 10 to 130+

Reply:
Got rid of many versions